src/javascript/_common/url.js

```diff
@@ -72,6 +72,24 @@ const Url = (() => {
         return static_host + path.replace(/(^\/)/g, '');
     };
 
+    /**
+     * @param {Object} new_params - Object with param-value pairs. To delete param, set value to null.
+     * @param {boolean} preserve_old - Should existing query parameters be preserved.
```

Comment:
for boolean the variable name should preferably start with `is_`, or in this case `should_`